### PR TITLE
Master api bpartner v2 tmp

### PR DESCRIPTION
**Pull request by cp-ps**

### CODE REVIEW

File: backend/de.metas.externalreference/src/main/java/de/metas/externalreference/bpartner/BPartnerValueExternalReferenceType.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.adempiere.exceptions.AdempiereException;
+import org.compiere.model.I_C_BPartner;
+
+import static de.metas.externalreference.model.X_S_ExternalReference.TYPE_BPartnerValue;
+
+@AllArgsConstructor
+@Getter
+public enum BPartnerValueExternalReferenceType implements IExternalReferenceType
```
Comment (metas-ts) [inline backend/de.metas.externalreference/src/main/java/de/metas/externalreference/bpartner/BPartnerValueExternalReferenceType.java:35]: why not have this enum-value in BPartnerExternalReferenceType?

File: backend/de.metas.externalreference/src/main/java/de/metas/externalreference/ExternalIdentifier.java [hunk 2]
```diff
@@ -35,6 +36,8 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import static de.metas.util.Check.isEmpty;
+
 @Value
 public class ExternalIdentifier
```
Comment (metas-ts) [inline backend/de.metas.externalreference/src/main/java/de/metas/externalreference/ExternalIdentifier.java:42]: please clarify the relation between this class and IdentifierString. If this class for v2 and identifierstring for v1?
but i found (just) one usage of IdentifierString in `de.metas.rest_api.v2.bpartner.BpartnerRestControllerTest#createOrUpdateBankAccount_create`

**Review by metas-ts — CHANGES_REQUESTED** [review]

File: backend/de.metas.externalreference/src/main/java/de/metas/externalreference/bpartner/BPartnerValueExternalReferenceType.java (hunk 1 above)
Comment (cif-ps) [reply backend/de.metas.externalreference/src/main/java/de/metas/externalreference/bpartner/BPartnerValueExternalReferenceType.java:35]: I will do the changes.

File: backend/de.metas.externalreference/src/main/java/de/metas/externalreference/ExternalIdentifier.java (hunk 2 above)
Comment (cif-ps) [reply backend/de.metas.externalreference/src/main/java/de/metas/externalreference/ExternalIdentifier.java:42]: I missed that, will fix it.

**Review by metas-ts — APPROVED** [review]

**Pull request merged**